oslo/torch/nn/modules/ngram_repeat_block.py

```diff
@@ -0,0 +1,122 @@
+# Originally from Microsoft Corporation.
```

Comment:
I think we should make the wrapper I said for this.

Reply:
Ok I'll try

Reply:
Add wrapper on `kernel_fusion_utils.py`